OpenRA.Mods.Common/Traits/Sellable.cs

```diff
@@ -57,5 +62,15 @@ public void Sell(Actor self)
 			else
 				self.QueueActivity(new Sell());
 		}
+
+		public bool IsVisible(Player forPlayer)
+		{
+			return forPlayer == self.Owner;
```

Comment:
This doesn't work as expected. I was able to see refund values on an AI player's building.